tests/check_prog.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+import argparse
+import pathlib
+import os
+import subprocess
+
+parser = argparse.ArgumentParser()
```

Comment:
I would take a look at the other scripts in https://github.com/p4gauntlet/gauntlet/blob/master/src/get_semantics.py#L180 for example.